applications/asset_tracker_v2/src/modules/location_module.c

```diff
@@ -30,14 +30,6 @@ BUILD_ASSERT(CONFIG_AT_MONITOR_HEAP_SIZE >= 1024,
 	    "CONFIG_AT_MONITOR_HEAP_SIZE must be >= 1024 to fit neighbor cell measurements "
 	    "and other notifications at the same time");
 
-/* Use a timeout of 90 seconds for GNSS search. */
-#define DATA_FETCH_TIMEOUT_GNSS_SEARCH 90
-
-/* Use a timeout of 11 seconds to accommodate for neighbour cell measurements
- * that can take up to 10.24 seconds.
- */
-#define DATA_FETCH_TIMEOUT_NEIGHBORHOOD_SEARCH 11
-
```

Comment:
Unrelated/Accidential change?

Reply:
Cleaning up old leftovers that haven't been used for a while.